tools/docker/scripts/test_python.sh

```diff
@@ -19,41 +19,48 @@ function run_selftests {
         echo "Selftesting $i"
```

Comment:
Sorry, I overlooked this one:
`"Selftesting $i"`  ->  `"Running $i"`

Reply:
done, thanks